### PR TITLE
Issue #463 Re-open Tig instantly in the last displayed view after exe…

### PR DESCRIPTION
**Pull request by DiegoPomares**
Implemented the > flag, I called it 'quick'  internally:

- I couldn't think of any tests to add :/
- The message (press Enter to continue) is still shown if ok in 'ok = io_run_fg(...)' is false (line 86@src/display.c), but maybe it is better to hide it as well even if ok is false. I'll leave that to you.
- Perhaps it would be better to use the quick mode for the merge tool as well (line 679@src/status.c), but I left it as it was. I'll also leave that to you.

### CODE REVIEW

**Comment by Alok** [comment]
Any chance of this getting merged soon/

**Comment by aesnyder** [comment]
Curious if this will be merged as well.

**Comment by DiegoPomares** [comment]
Dunno @aesnyder @alok :man_shrugging:
BTW guys since you seem interested, what do you think about my original remarks about not showing the _press Enter to continue_ message when the command exits with a non 0 status, and for the merger tool?

**Comment by alok** [comment]
@DiegoPomares I'm not sure about the nonzero exit code case, but for merge I think it's fine.

**Comment by DiegoPomares** [comment]
@alok That was my idea initially, thanks for the feedback :smile:

**Comment by alok** [comment]
@jonas any change of this getting added?

File: src/status.c [hunk 1]
```diff
@@ -676,7 +676,7 @@ open_mergetool(const char *file)
 {
 	const char *mergetool_argv[] = { "git", "mergetool", file, NULL };
 
-	open_external_viewer(mergetool_argv, repo.cdup, false, true, false, true, "");
+	open_external_viewer(mergetool_argv, repo.cdup, false, true, false, true, true, "");
```
Comment (jonas) [inline src/status.c]: Not sure what the motivation is for using the quick mode here. I'll change it to the original behavior.

**Comment by jonas** [comment]
Thanks a lot @DiegoPomares ! 🎉

**Pull request merged**